### DISCUSSION TITLE
Windows - Starting the cmd at the current folder path, the fast way

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#3711] [General]
I wanted to share a neat trick with you that i wasn't aware of.

When you open a folder, and type cmd in the adress line of the folder, then the command line opens at this path. 

This solves one of the inconveniences at Windows for me regarding the command line. In Linux you right click and choose open terminal, and you are at the right path already. At Windows i was always at copying the path over and dealing with the drive letters. Not longer :)

![one](https://github.com/Bforartists/Bforartists/assets/13919702/2356f6f5-0c30-4112-bed8-9048abb0a5d7)

![two](https://github.com/Bforartists/Bforartists/assets/13919702/d7ea4240-c52c-41f6-83e8-2420fc55fbe2)

![three](https://github.com/Bforartists/Bforartists/assets/13919702/90ac27c2-fec1-4377-8733-b1df0731d14c)

**Comment by Draise14** [comment]
If you are in Windows 11, you can also right click and "Open in Terminal" like you would in Linux. 

![image](https://github.com/Bforartists/Bforartists/assets/35357409/b26c76dc-6ad7-4283-b654-9f4e26d599a5)

**Comment by ReinerBforartists** [comment]
Not here ^^
Are you on Windows 11 already? I still use Win 10.

**Reply by Draise14** [reply]
Yeah, on W11, upgraded earlier this year when I got my new rig - so thought to bite the bullet since W10 will phase out support later this year or next. I had to purchase Stardock Start11 though, to get the start menu I liked back (full screen, tiles, my whole ecosystem folders and all). StartAllBack also, though freemium, also got the W10 ribbon back too - and it gets the old context menus back too without registry changes. W10 has been a good one - but the other W11 settings and other bits of polish is nice, same with window snapping/organizing. It technically also has a linux sub-system, but I haven't figured it out yet. 

Edit: I also love the tabbed terminal. 

![WindowsTerminal_woCi1WpE1F](https://github.com/Bforartists/Bforartists/assets/35357409/97941b8d-1a12-4055-84e8-bf208c5d0ca3)

**Reply by ReinerBforartists** [reply]
Yeah that looks nice for sure :)

I still want to wait a while. Never touch a running system when you don't have to. And i still don't need something from the Win 11 eco system. Sooner or later i have to switch i guess. But for now i stay as long as possible.
